android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -773,7 +838,13 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
         break;
       case COMMAND_INJECT_JAVASCRIPT:
         RNCWebView reactWebView = (RNCWebView) root;
-        reactWebView.evaluateJavascriptWithFallback(args.getString(0));
+        if(!TextUtils.isEmpty(args.getString(0))){
```

Comment:
I'm not sure what this does but you should not inject javascript like to setTextZoom anyway